src/main/java/ca/corefacility/bioinformatics/irida/web/controller/api/samples/RESTSampleSequenceFilesController.java

```diff
@@ -400,29 +401,33 @@ public ModelMap addNewSequenceFileToSample(@PathVariable Long sampleId, @Request
 		file.transferTo(target.toFile());
```

Comment:
Can you move this stuff inside the `try` as well?  That way if the transfer dies it can still clean up the tmp files.

Reply:
I've added everything that can be put in the `try` in the `try` block now. the `temp` and `target` paths need to stay outside of the `try` block so that the `finally` can always access them.